views/templates/hook/front/carrier-phone-number.tpl

```diff
@@ -8,18 +8,16 @@
                     class="chosen-select form-control form-control-chosen"
                     name="dpd-phone-area"
                     required="required">
-                    {html_options options=$dpdPhoneArea selected=$contextPrefix}
+                {html_options options=$dpdPhoneArea selected=$contextPrefix}
             </select>
             <div class="control-label dpd-input-placeholder dpd-select-placeholder">
                 {l s='Code' mod='dpdbaltics'}
             </div>
         </div>
-
         <div class="col-lg-4 col-8 col-sm-12 dpd-input-wrapper{if isset($dpdPhone) && $dpdPhone} hasValue{/if} small-padding-sm-left">
-            <input name="dpd-phone" type="text" required class="form-control" {if isset($dpdPhone) && $dpdPhone}value="{$dpdPhone}"{/if}>
+            <input name="dpd-phone" id={$carrierID} type="text" class="form-control" {if isset($dpdPhone) && $dpdPhone}value="{$dpdPhone}"{/if}>
```

Comment:
dont think its good idea to set id just carrier id, do something like dpd-carrier-{$id}
what if somewhere else something will do id=5 and it will be the same?